massa-node/base_config/openrpc.json

```diff
@@ -267,7 +270,7 @@
             "params": [
                 {
                     "name": "endorsementId",
-                    "description": "Must be an endorsement id",
+                    "description": "Must be an endorsement(s) id(s)",
```

Comment:
Maybe better: "Need to provide at least one valid endorsement id"

Reply:
done